configs/encapsulate_in_http1_connect.yaml

```diff
@@ -31,8 +31,11 @@ static_resources:
     - name: cluster_0
       connect_timeout: 5s
       # This ensures HTTP/1.1 CONNECT is used for establishing the tunnel.
-      http_protocol_options:
-        {}
+      typed_extension_protocol_options:
+        envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+          "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+          explicit_http_config:
```

Comment:
This is related to my comment above, but if the default is to use HTTP/1 and we force one of the types to be set, would it be better to do:

```
explicit_http_config: {}
```

?

Reply:
well, the explicit http_protocol_options was already not needed before.  I think it was put in to emphasize the differences between HTTP/1 and HTTP/2 encapsulate config, though it arguably could be used to highlight the defaults instead of highlighting explicit config.